docs/contributing/test-data-models-overview.md

```diff
@@ -0,0 +1,250 @@
+# Test data models overview
+
+A test data model is set of helper objects commercetools API consumers can use during writing their own tests in order to mock the data returned by our services.
+
+These helpers are flexible enough to represent different types of responses from different APIs and with different values.
+
+## Public API
+
+Each test data model has the same API which consists on:
+
+- A Rest builder
+- A Graphql builder
+- Typescript types
+
+A `Builder` is the object you would be interacting with as a consumer and it has only two functions:
+
+- `random`: exposes a fluent API which allows to build a customized version of the data model
+- `presets`: predefined versions of the data model
+
+This is how it looks like the entry point of a data model package:
+
+```ts
+import { RestModelBuilder, GraphqlModelBuilder } from './builders';
+import * as GeometryPresets from './presets';
+
+export * from './types';
+
+export const GeometryRest = {
+  random: RestModelBuilder,
+  presets: GeometryPresets.restPresets,
+};
+
+export const GeometryGraphql = {
```

Comment:
How are we treating the `q` in other places here? Should it be: GeometryGraphQl. No energy for strong feelings towards it.

Reply:
Sorry but I'm not sure what you mean by "other places".
Currently, this is how I set it up in the CLI generator so I'm consistent with it.
Just let me know if you prefer another format so I can adjust it.

Reply:
If I get to pick: let's use `GraphQL` in writing and `GraphQl` in code. That's the way I've seen people use it mostly.

Reply:
It there is a common pattern how to capitalise the term then let's continue to use it. 
If we have inconsistency already then I'd prefer to to with GeometryGraphQl, GraphQlModelBuilder and so on in code. In text we should use GraphQL. 

I am happy to do this as a follow up of this PR.

Reply:
I think we don't currently have inconsistencies as we're using the `Graphql` version in code everywhere I can think of, even in some existing code ([reference](https://github.com/commercetools/test-data/blob/main/models/commons/src/localized-string/helpers.ts#L18)).

I agree using `GraphQL` in docs, but don't see much improvement in changing the code.

Reply:
Thanks. Sorry for the detail here and assumption of inconsistencies. Then it's all good.